tests/jobs/test_local_task_job.py

```diff
@@ -715,6 +699,8 @@ def test_fast_follow(
                 for upstream, downstream in dependencies.items():
                     dag.set_dependency(upstream, downstream)
 
+            dag_maker.make_dagmodel()
```

Comment:
This shouldn't have to be called -- I think we should _always_ create the DagModel, as my high-level goal with asking for this fixture is to make tests more representative of _How_ Airflow is when running, specifically that the SerializedDag and DagModel rows are always present for a DAG.

And in the _rare_ cases where we don't want either of these we could then delete them in the tests.

Reply:
The problem was on separating the arguments that DagModel takes from the ones that DAG takes.